src/librustc_trans_utils/assert_module_sources.rs

```diff
@@ -38,7 +38,7 @@ const CFG: &'static str = "cfg";
 #[derive(Debug, PartialEq, Clone, Copy)]
 enum Disposition { Reused, Translated }
 
-pub(crate) fn assert_module_sources<'a, 'tcx>(tcx: TyCtxt<'a, 'tcx, 'tcx>) {
+pub fn assert_module_sources<'a, 'tcx>(tcx: TyCtxt<'a, 'tcx, 'tcx>) {
```

Comment:
Shouldn't this be in `rustc_incremental`?

Reply:
Done locally